.NET/Microsoft.Recognizers.Text.NumberWithUnit/Extractors/DinoComparer.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Text.Matcher;
+
+namespace Microsoft.Recognizers.Text.NumberWithUnit
+{
+    public class DinoComparer : IComparer<string>
```

Comment:
Nit: @enzocanoo, do you remember what this comparer does differently? We can perhaps give it a better name. It seems it was added by you back in 7109d76e.
Not a major issue. I'll merge as is, but let's see if we can give it a clearer name.

Reply:
Perhaps also move it out of /Extractors to /Utilities as it's potentially useful elsewhere.